mod-consortia/src/main/resources/thunderjet/mod-consortia/features/consortia-user-type-update.feature

```diff
@@ -87,4 +87,73 @@ Feature: Consortia User type Update tests
     And headers {'x-okapi-tenant':'#(centralTenant)', 'x-okapi-token':'#(okapitoken)'}
     And retry until response.totalRecords == 0
     When method GET
-    Then status 200
\ No newline at end of file
+    Then status 200
+
+
+  @Positive
+  Scenario: Verify that two system users can be created in different tenant
+    # Create two system user with username 'user1'
+    Given path 'users'
+    And headers {'x-okapi-tenant':'#(universityTenant)', 'x-okapi-token':'#(okapitoken)'}
+    And request
+    """
+    {
+      "id":"1efc2b3e-76de-41dc-a3c3-a5ef90e33483",
+      "username": 'user1',
+      "active":true,
+      "personal": {"firstName":"User 1","lastName":'User 1'},
+      "type": "system"
+    }
+    """
+    When method POST
+    Then status 201
+
+    * call pause 1000
+
+    Given path 'users'
+    And headers {'x-okapi-tenant':'#(collegeTenant)', 'x-okapi-token':'#(okapitoken)'}
+    And request
+    """
+    {
+      "id":"b5fc7128-b783-4820-9a6b-3d4b35ebeff9",
+      "username": 'user1',
+      "active": true,
+      "personal": {"firstName":"User 1","lastName":'User 1'},
+      "type": "system"
+    }
+    """
+    When method POST
+    Then status 201
+
```

Comment:
please add here comment that we are creating 2 staff users where username validation accross tenants sshould exist

Reply:
Thanks, added